posts/2024-03-08-Rustup-1.27.0.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: post
+title: "Announcing Rustup 1.27.0"
+author: The Rustup Team
+---
+
+The rustup team is happy to announce the release of rustup version 1.27.0.
+[Rustup][install] is the recommended tool to install [Rust][rust], a programming language that is empowering everyone to build reliable and efficient software.
+
+If you have a previous version of rustup installed, getting rustup 1.27.0 is as easy as stopping any programs which may be using Rustup (e.g. closing your IDE) and running:
+
+```console
+$ rustup self update
+```
+
+Rustup will also automatically update itself at the end of a normal toolchain update:
+
+```console
+$ rustup update
+```
+
+If you don't have it already, you can [get rustup][install] from the appropriate page on our website.
+
+[rust]: https://www.rust-lang.org
+[install]: https://rustup.rs
+
+## What's new in rustup 1.27.0
+
+This long-awaited Rustup release has gathered all the new features and fixes since April 2023. These changes include improvements in Rustup's maintainability, user experience, compatibility and documentation quality.
+
+Also, it's worth mentioning that Dirkjan Ochtman (djc) and rami3l (rami3l) have joined the team and are coordinating this new release.
+
+At the same time, we have granted Daniel Silverstone (kinnison) and 二手掉包工程师 (hi-rustin) their well-deserved alumni status in this release cycle.
+Kudos for your contributions over the years and your continuous guidance on maintaining the project!
+
+The headlines for this release are:
+
+1. Basic support for fish shell has been added.
+   If you're using `fish`, PATH configs for your Rustup installation will be added automatically from now on.
+
+   _Please note that this will only take effect on installation,
+   so you will need to rerun `rustup-init` if you already have installed Rustup on your machine._
```

Comment:
`rustup-init` isn't available to me (zsh: command not found: rustup-init) with rustup installed. Should it be? Should we link this to rustup.rs or something else instead?

Reply:
@Mark-Simulacrum Thanks for your suggestion! I elaborated a bit on how exactly to reinstall Rustup in different scenarios.